src/bot.ts

```diff
@@ -77,6 +82,25 @@ export const robot = (app: Probot) => {
           return [...new Set(allOwners)];
         }
 
+        async function isProviderAlreadyBranded(apiUrl: string, identity: string, provider: string): Promise<{
+          existing: boolean,
+          isUpdate?: boolean
+        }> {
+          const providerInfo = await axios.get(`${apiUrl}/providers/${provider}`);
```

Comment:
Here you can include try/catch in case of API is down

Reply:
added